adding-sessions.md

```diff
@@ -61,34 +134,80 @@ Note that we can't just write `params[:username]`; we must write `params[:user][
   "commit"=>"Submit"
 }
 ```
+> Note that `username` and `password` exist inside of a `user` hash.
+
+## 5. Let Your User Sign Out
+
+Let's give the user the option to sign out. Let's start by adding a link to navigation that will eventually allow the user to do this. Place the following code anywhere inside of the `<nav>` element...
+
+```html
+<!-- application.html.erb -->
+
+<%= link_to "Sign Out", session_path, method: :delete %>
+```
 
-## [5. Let your user sign out](https://github.com/ga-wdi-exercises/tunr_rails_users/pull/3/files#diff-d5241d488259f32ecbe2f636133e5ddaR25)
+This link will correspond to a `destroy` method that we will define in our Sessions controller. In this method, we need to remove `session[:user_id]` or set it to `nil`. That way, the application thinks nobody is signed in.
 
-To "sign out" the user, we simply need to remove `session[:user_id]` or set it to `nil`. Then, the next time the `before_action` runs, `@current_user` will also be nil.
+```rb
+def destroy
+  reset_session   # Delete all sessions
+  puts "You're signed out!"
+  redirect_to :root   # Send the user back to the homepage
+end
+```
 
-`reset_session` is a shortcut to remove *all* session variables at once.
+`reset_session` is a shortcut that comes packaged with Rails that is used to remove *all* session variables at once.
 
 Try signing in and signing out now!
 
-## [6. Be able to access the current user](https://github.com/ga-wdi-exercises/tunr_rails_users/pull/3/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR5)
+## 6. Be Able To Access The Current User
+
+Wouldn't it be nice if, when a user is signed in, we had a shorthand for accessing that user. We'll do that by defining our own `@current_user` instance variable that is accessible throughout the application.
```

Comment:
normally i would say this would conflate the lesson, but we're introducing before_action here anyway.